database-plugins/src/main/java/io/cdap/plugin/DBRecord.java

```diff
@@ -240,14 +249,20 @@ private void writeToDB(PreparedStatement stmt, Schema.Field field, int fieldInde
       switch (fieldLogicalType) {
         case DATE:
           stmt.setDate(sqlIndex, Date.valueOf(record.getDate(fieldName)));
+          bytesWritten += ((Long) record.get(fieldName)).BYTES;
```

Comment:
Long.BYTES is a static constant, shouldn't be accessing it via instance.

Reply:
fixed